freqtrade/exchange/exchange_utils.py

```diff
@@ -11,13 +11,26 @@
 
 from freqtrade.exchange.common import (BAD_EXCHANGES, EXCHANGE_HAS_OPTIONAL, EXCHANGE_HAS_REQUIRED,
                                        SUPPORTED_EXCHANGES)
+from freqtrade.exchange.exchange_utils_timeframe import (timeframe_to_minutes, timeframe_to_msecs,
+                                                         timeframe_to_next_date,
+                                                         timeframe_to_prev_date,
+                                                         timeframe_to_resample_freq,
+                                                         timeframe_to_seconds)
```

Comment:
Well **if** we remove them from this file - then we shouldn't import them here unless they're actually used (i doubt they are used here ... at least not all of them).

instead - imports should be adjusted accordingly if necessary - which ideally should only be in exchange.py - and eventually in exchange/__init__.py ... otherwise the imports were wrong anyway.

Reply:
on it